skipper.go

```diff
@@ -1145,6 +1149,24 @@ func (o *Options) tlsConfig(cr *certregistry.CertRegistry) (*tls.Config, error)
 	return config, nil
 }
 
+func (o *Options) tracerInstance() (ot.Tracer, error) {
+	if o.Tracer != nil {
+		return o.Tracer, nil
+	}
+
+	if len(o.OpenTracing) > 0 {
+		return tracing.InitTracer(o.OpenTracing)
+	} else {
+		// always have a tracer available, so filter authors can rely on the
+		// existence of a tracer
+		tracer, _ := tracing.LoadTracingPlugin(o.PluginDirs, []string{"noop"})
```

Comment:
The default option is to use `noop` so we do not need to create ot.NoopTracer{} so lets not swallow the error but simply `return tracing.LoadTracingPlugin(o.PluginDirs, []string{"noop"})`

Reply:
Lets do this:
```go
if len(o.OpenTracing) > 0 {
  return tracing.InitTracer(o.OpenTracing)
} else {
  // always have a tracer available, so filter authors can rely on the
  // existence of a tracer
  tracer, err := tracing.LoadTracingPlugin(o.PluginDirs, []string{"noop"})
  if err != nil {
    return err
  } else if tracer == nil {
    // LoadTracingPlugin unfortunately may return nil tracer
    return fmt.Errorf("failed to load tracing plugin")
  }
  return tracer, nil
```